src/stars.css

```diff
@@ -0,0 +1,91 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  margin-bottom: 4px;
+}
+
+/* .stars__star:last-child {
+  margin-right: 0;
+} */
```

Comment:
remove the comments

Reply:
I have a doubt here. In figma there is no margin at right side of the block. I implemented this margin-right: 0, but the backstop didn't passed the test comparisson.